auth0/v3/management/auth0.py

```diff
@@ -40,6 +41,7 @@ def __init__(self, domain, token):
         self.logs = Logs(domain, token)
         self.resource_servers = ResourceServers(domain, token)
         self.rules = Rules(domain, token)
+        self.rules_configs = RulesConfigs(domain, token)
         self.stats = Stats(domain, token)
```

Comment:
`Custom Domains` and `Grants` also should be added here. Please try to keep the alphabetical order 💯